src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/MarkDuplicatesDataflowIntegrationTest.java

```diff
@@ -25,7 +25,15 @@
         };
     }
 
-    @Test(groups = "dataflow", dataProvider = "md")
+    @DataProvider(name="md_optical")
+    public Object[][] md_optical() {
+        return new Object[][] {
+          {new File(MarkDuplicatesIntegrationTest.TEST_DATA_DIR, "optical_dupes.bam"), 4, 2},
+          /*{new File(MarkDuplicatesIntegrationTest.TEST_DATA_DIR, "optical_dupes_casava.bam"), 4, 2},*/
```

Comment:
Had to fix something in OpticalDuplicateFinder. Fixed it, and the test is in now.